kronos-java/src/main/java/com/lyft/kronos/sntp/Dns.kt

```diff
@@ -0,0 +1,20 @@
+package com.lyft.kronos.sntp
+
+import java.net.InetAddress
+import java.net.UnknownHostException
+
+internal interface Dns {
+
+    fun resolve(host: String): Array<out InetAddress>
+
+    class Impl : Dns {
+
+        override fun resolve(host: String) = try {
+            InetAddress.getAllByName(host).orEmpty()
+        } catch (e: SecurityException) {
+            emptyArray()
+        } catch (e: UnknownHostException) {
+            emptyArray()
+        }
+    }
+}
```

Comment:
In comparison to `internal.ntp.DnsResolver` it hides exception handling from consumers (instead of enforcing checked exceptions) and supports multiple addresses for a host. For example, pool NTP servers have multiple IP addresses:

```console
$ nslookup "0.pool.ntp.org"

Address: 178.124.134.106
Address: 31.130.200.2
Address: 194.158.196.171
Address: 82.209.245.153
```

Also there are some tests for it.